HTML/index.html

```diff
@@ -62,7 +68,7 @@ <h5 class="subtitle grid_6 test">POSTED ON 15 DEC 2013 / DESSERTS<h5>
 
         <section class="recipe test">
           <img class="list" src="file:///Users/jkoehnke/Downloads/challenge/list-bg.png" alt="List" title="List">
```

Comment:
It would be better to put this in the css as a background-image of .recipe

Here's a [good overview of background](https://css-tricks.com/almanac/properties/b/background/)